gremlin-javascript/example/example.js

```diff
@@ -0,0 +1,143 @@
+/*
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+*/
+
+// Common imports as listed at reference/#gremlin-javascript-imports
+const gremlin = require('gremlin');
+const traversal = gremlin.process.AnonymousTraversalSource.traversal;
+const __ = gremlin.process.statics;
+const DriverRemoteConnection = gremlin.driver.DriverRemoteConnection;
+const column = gremlin.process.column;
+const direction = gremlin.process.direction;
+const Direction = {
+  BOTH: direction.both,
+  IN: direction.in,
+  OUT: direction.out,
+  from_: direction.out,
+  to: direction.in,
+}
+const p = gremlin.process.P;
+const textp = gremlin.process.TextP;
+const pick = gremlin.process.pick;
+const pop = gremlin.process.pop;
+const order = gremlin.process.order;
+const scope = gremlin.process.scope;
+const t = gremlin.process.t;
+const cardinality = gremlin.process.cardinality;
+const CardinalityValue = gremlin.process.CardinalityValue;
+const serializer = gremlin.structure.io.graphserializer;
+const testing = Direction.to;
+
+async function main() {
+    await connectionExample();
+    await basicGremlinExample();
+    await modernTraversalExample();
+}
+
+
+
+async function connectionExample() {
+    // Connecting to the server
+    let dc = new DriverRemoteConnection('ws://localhost:8182/gremlin');
+    let g = traversal().withRemote(dc);
+
+    // Connecting and customizing configurations
+    g = traversal().withRemote(new DriverRemoteConnection('ws://localhost:8182/gremlin', {
+        mimeType: 'application/vnd.gremlin-v3.0+json',
+        reader: serializer,
+        writer: serializer,
+        rejectUnauthorized: false,
+        traversalSource: 'g',
+    }));
+    
+    // Cleanup
+    await dc.close();
```

Comment:
could you add simple query here to verify connection parameters are valid?

Reply:
Added simple addV and count query